api/src/test/java/org/openmrs/api/AdministrationServiceTest.java

```diff
@@ -289,218 +323,219 @@ public void updateGlobalProperty_shouldUpdateAGlobalPropertyWhoseTypedvalueIsHan
-		
-		Assert.assertEquals("new prop value1", Context.getAdministrationService().getGlobalProperty("new prop1"));
-		Assert.assertEquals("new prop value2", Context.getAdministrationService().getGlobalProperty("new prop2"));
+
+		assertEquals("new prop value1", Context.getAdministrationService().getGlobalProperty("new prop1"));
+		assertEquals("new prop value2", Context.getAdministrationService().getGlobalProperty("new prop2"));
 	}
-	
+
 	/**
 	 * @see AdministrationService#saveGlobalProperties(List)
 	 */
 	@Test
 	public void saveGlobalProperties_shouldAssignUuidToAllNewProperties() {
 		// get the current global properties
 		List<GlobalProperty> globalProperties = Context.getAdministrationService().getAllGlobalProperties();
-		
+
 		// and now add a new one to it and save it
 		globalProperties.add(new GlobalProperty("new prop", "new prop value", "desc"));
 		Context.getAdministrationService().saveGlobalProperties(globalProperties);
-		
-		Assert.assertNotNull(Context.getAdministrationService().getGlobalPropertyObject("new prop").getUuid());
+
+		assertNotNull(Context.getAdministrationService().getGlobalPropertyObject("new prop").getUuid());
 	}
-	
+
 	/**
 	 * @see AdministrationService#getAllGlobalProperties()
 	 */
 	@Test
 	public void getAllGlobalProperties_shouldReturnAllGlobalPropertiesInTheDatabase() {
 		executeDataSet(ADMIN_INITIAL_DATA_XML);
-		Assert.assertEquals(21, Context.getAdministrationService().getAllGlobalProperties().size());
+		assertEquals(21, Context.getAdministrationService().getAllGlobalProperties().size());
 	}
-	
+
 	/**
 	 * @see AdministrationService#getAllowedLocales()
 	 */
 	@Test
 	public void getAllowedLocales_shouldReturnAtLeastOneLocaleIfNoLocalesDefinedInDatabaseYet() {
-		Assert.assertTrue(Context.getAdministrationService().getAllowedLocales().size() > 0);
+		assertTrue(Context.getAdministrationService().getAllowedLocales().size() > 0);
 	}
-	
+
 	/**
 	 * @see AdministrationService#getGlobalPropertyObject(String)
 	 */
 	@Test
 	public void getGlobalPropertyObject_shouldReturnNullWhenNoGlobalPropertyMatchGivenPropertyName() {
 		executeDataSet(ADMIN_INITIAL_DATA_XML);
-		Assert.assertNull(Context.getAdministrationService().getGlobalPropertyObject("magicResistSkill"));
+		assertNull(Context.getAdministrationService().getGlobalPropertyObject("magicResistSkill"));
 	}
-	
+
 	/**
 	 * @see AdministrationService#getImplementationId()
 	 */
 	@Test
 	public void getImplementationId_shouldReturnNullIfNoImplementationIdIsDefinedYet() {
 		executeDataSet(ADMIN_INITIAL_DATA_XML);
-		Assert.assertNull(Context.getAdministrationService().getImplementationId());
-	}
-	
-	/**
-	 * @see AdministrationService#getPresentationLocales()
-	 */
-	@Test
-	@Ignore
-	//TODO: This test fails for some reason
-	public void getPresentationLocales_shouldReturnAtLeastOneLocaleIfNoLocalesDefinedInDatabaseYet() {
-		Assert.assertTrue(Context.getAdministrationService().getPresentationLocales().size() > 0);
```

Comment:
you deleted the test. did you try to fix it?

Reply:
I saw that the code was covered. Can you please explain why this test is relevant?